### DISCUSSION TITLE
Suggestion: Autodetection tool for bitrate

### DISCUSSION THREAD

**Discussion by goliv04053** [jcorporation/webradiodb#855] [General]
I am having some issues with manually setting bitrate information because there are some radios that i can't deduce or to see fully that it's that bitrate that I input there. So, i think that would be amazing to facilitate and implement an autodetect function. Radio-Browser does it on stream check. @jcorporation

**Comment by jcorporation** [comment]
This is already implemented. Leave it empty and it should be populated through a GitHub action that runs each night.

**Reply by goliv04053** [reply]
Oh.... I didn't knew it. Okay then. Some which were added manually (the data refered), should I delete the part and it will be added automatically? I mean, sending a edit request on issues/website edit function removing the data
